operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/retry/RetryingGradually.java

```diff
@@ -0,0 +1,28 @@
+package io.javaoperatorsdk.operator.processing.retry;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+public @interface RetryingGradually {
```

Comment:
Maybe `GradualRetry`, but this implementation is such generic, that maybe just `Retry` ?